learn/hybrid-mobile/export-cordova-zip.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Export Cordova zip"
+id: ""
+---
+---
+
+A Cordova zip will be used to create a mobile application. WaveMaker allows you to export a cordova zip from a *mobile project* so that
+you can use it to generate a *mobile application*.
+
+Following are the two approaches which can be followed to create a cordova zip 
+
+## 1. Using WaveMaker studio
+
+1. In the top right menu bar of WaveMaker Studio, export menu is present.  Click on 'Download Cordova zip' menu item.
```

Comment:
I think "Download Cordova zip" should be renamed to "Project As Cordova Zip" similar to how the other ones are named.

Reply:
@PatnalaPriyanka is working on the UI change. Will update the document in next minor release